L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -25,8 +25,8 @@
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('Configuration.StandardSequences.MagneticField_cff')
-process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
-process.load('Configuration.Geometry.GeometryExtended2026D49_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D77Reco_cff')
```

Comment:
I think this is fine for the moment, however it will be very short as we plan to remove as you aware of. Do you want to move it now?

Reply:
We will move later, so this PR can go first independently